lib/utils.js

```diff
@@ -5,19 +5,38 @@ const exec = require('child_process').exec;
 const qs = require('qs');
 const crypto = require('crypto');
 
-const hasOwn = {}.hasOwnProperty;
-const isPlainObject = require('lodash.isplainobject');
+const hasOwn = Function.bind.call(
+  Function.call,
+  Object.prototype.hasOwnProperty
+);
 
-const OPTIONS_KEYS = [
+const OPTIONS_KEYS = new Set([
   'api_key',
   'idempotency_key',
   'stripe_account',
   'stripe_version',
-];
+]);
 
 const utils = (module.exports = {
-  isOptionsHash: (o) => {
-    return isPlainObject(o) && OPTIONS_KEYS.some((key) => hasOwn.call(o, key));
+  isSerializableObject(obj) {
+    if (typeof obj !== 'object') {
+      return false;
+    }
+    try {
+      const str = JSON.stringify(obj);
+      return str === JSON.stringify(JSON.parse(str));
+    } catch (_) {
+      return false;
+    }
+  },
+
+  isOptionsHash(o) {
+    return (
+      this.isSerializableObject(o) &&
+      Object.keys(o).filter(
+        (p) => OPTIONS_KEYS.has(p) && typeof o[p] === 'string'
```

Comment:
We are doing here better job that original check did, because we also make sure the property value is `string`

Reply:
Hmm, I'd rather not do that – if someone passes `{idempotency_key: null}` I'd rather it continue to get picked up as an option, so they can understand what's happening.